README.rst

```diff
@@ -428,13 +428,13 @@ it is not sent if the response was not successful.
 Default Alerts / Notifications
 ------------------------------
 
-+-----------------------+------------------------------------------------------------------------------+
-| Notification Types    | Uses                                                                         |
-+-----------------------+------------------------------------------------------------------------------+
-| ``threshold_crossed`` | Used for creating notification when a metric exceeds threshold value.        |
-+-----------------------+------------------------------------------------------------------------------+
-| ``under_threshold``   | Used for creating notification when a metric falls behind threshold value.   |
-+-----------------------+------------------------------------------------------------------------------+
++----------------------------+---------------------------------------------------------------------------------+
+|    Notification Types      | Uses                                                                            |
++----------------------------+---------------------------------------------------------------------------------+
+| ``alert_settings_crossed`` | Used for creating notification when a metric exceeds alert settings value.      |
```

Comment:
AlertSettings.value is still indicated as the threshold value so I think we can keep using this form because is clear, I will rename this.